interni/norme-di-progetto/components/processi-di-supporto/validazione.tex

```diff
@@ -13,23 +12,47 @@ \subsubsection{Descrizione}%
 
 Dovendo verificare se il prodotto finale soddisfa i requisiti, anche il processo di validazione fa uso dell'analisi dinamica del codice e quindi di test ad hoc per dimostrare la correttezza del prodotto.
 
-\subsubsection{Piano di validazione}%
-\label{subs:piano_di_validazione}
+\subsubsection{Attività}%
+\label{subs:attivita}
 
-Il piano di validazione comprende tutte le attività del processo di validazione:
+Il processo comprende due attività principali:
 
+\begin{itemize}
+\item implementazione del processo
+\item validazione.
+\end{itemize}
+
+\paragraph{Implementazione del processo}%
+\label{subs:validazione/implementazione}
+Un piano di implementazione dovrebbe includere almeno:
 \begin{itemize}
   \item Scelta dell'oggetto di validazione.
   \item Decisione dei compiti da eseguire per realizzare la validazione.
   \item Determinazione delle risorse e delle responsabilità.
   \item Procedure per fornire al cliente i report di validazione.
 \end{itemize}
 
-\subsubsection{Metriche}%
-\label{subs:validazione/metriche}
-
-% TODO scrivi metriche (copertura dei requisiti?)
+\paragraph{Validazione}
+L'attività di validazione consiste nel preparare test sui requisiti e analizzare i loro risultati.
+
+\paragraph{Test di accettazione}
+I test di accettazione vengono effettuati in presenza del proponente in modo da verificare se sono stati soddisfatti tutti i requisiti espliciti o impliciti del prodotto.
```

Comment:
Modificherei "in modo da verificare se sono" -> "in modo da verificare congiuntamente che siano"